{"name": "Test login with invalid username", "status": "passed", "description": "This test verifies that an error message is displayed when logging in with an invalid username.", "attachments": [{"name": "Invalid Username - Error Message", "source": "0671076d-d045-4eab-8191-007f1193e02e-attachment.png", "type": "image/png"}], "start": 1740395854772, "stop": 1740395858438, "uuid": "46280e90-f0ac-44d9-baa1-5896e04d840d", "historyId": "8dfefa9403d2f0167e5ccac7cf061037", "testCaseId": "8dfefa9403d2f0167e5ccac7cf061037", "fullName": "test_form_authentication.TestFormAuthentication#test_invalid_username", "labels": [{"name": "feature", "value": "Form Authentication"}, {"name": "story", "value": "Invalid Login"}, {"name": "suite", "value": "test_form_authentication"}, {"name": "subSuite", "value": "TestFormAuthentication"}, {"name": "host", "value": "Jonathan"}, {"name": "thread", "value": "13040-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_form_authentication"}]}